ReactAndroid/src/main/java/com/facebook/react/modules/network/NetworkingModule.java

```diff
@@ -186,38 +194,38 @@ public void sendRequest(
     }
 
     mClient.newCall(requestBuilder.build()).enqueue(
-        new Callback() {
-          @Override
-          public void onFailure(Request request, IOException e) {
-            if (mShuttingDown) {
-              return;
-            }
-            onRequestError(requestId, e.getMessage());
-          }
-
-          @Override
-          public void onResponse(Response response) throws IOException {
-            if (mShuttingDown) {
-              return;
-            }
-
-            // Before we touch the body send headers to JS
-            onResponseReceived(requestId, response);
-
-            ResponseBody responseBody = response.body();
-            try {
-              if (useIncrementalUpdates) {
-                readWithProgress(requestId, responseBody);
-                onRequestSuccess(requestId);
-              } else {
-                onDataReceived(requestId, responseBody.string());
-                onRequestSuccess(requestId);
+            new Callback() {
+              @Override
+              public void onFailure(Request request, IOException e) {
+                if (mShuttingDown) {
+                  return;
+                }
+                onRequestError(requestId, e.getMessage());
               }
-            } catch (IOException e) {
-              onRequestError(requestId, e.getMessage());
-            }
-          }
-        });
+
+              @Override
+              public void onResponse(Response response) throws IOException {
+                if (mShuttingDown) {
+                  return;
+                }
+
+                // Before we touch the body send headers to JS
+                onResponseReceived(requestId, response);
+
+                ResponseBody responseBody = response.body();
+                try {
+                  if (useIncrementalUpdates) {
+                    readWithProgress(requestId, responseBody);
+                    onRequestSuccess(requestId);
+                  } else {
+                    onDataReceived(requestId, responseBody.string());
+                    onRequestSuccess(requestId);
+                  }
+                } catch (IOException e) {
+                  onRequestError(requestId, e.getMessage());
+                }
+              }
+            });
```

Comment:
Why rewrite all of this? Is it just indentation? Can you keep it focused only on the actual changes? Doesn't look like you're changing `onFailure`.